junit-jupiter-api/src/main/kotlin/org/junit/jupiter/api/Assertions.kt

```diff
@@ -16,6 +17,24 @@ import org.junit.jupiter.api.function.Executable
 import java.util.function.Supplier
 import java.util.stream.Stream
 
+/**
+ * @see Assertions.fail
+ */
+fun fail(message: String?, throwable: Throwable? = null): Nothing =
+    Assertions.fail<Nothing>(message, throwable)
+
+/**
+ * @see Assertions.fail
+ */
+fun fail(message: (() -> String)?): Nothing =
+    Assertions.fail<Nothing>(message)
+
+/**
+ * @see Assertions.fail
+ */
+fun fail(throwable: Throwable): Nothing =
```

Comment:
Not really certain if this should be specified as nullable. I guess it could be? Maybe?

Reply:
Yes, `throwable` may be null.

Reply:
I should change this to `fun fail(throwable: Throwable?): Nothing` then. Will do.